app/services/stores/user_store.rb

```diff
@@ -13,6 +13,7 @@ def store_keys
     private
 
     def identifier_id
+      # TECH DEBT: [Stores::UserStore] This is error prone
       identifier_model.id
```

Comment:
Just trying to understand the issue a little more but why isn't this following the same format as the others?

Reply:
I think it was leftovers.